src/main/java/org/springframework/data/neo4j/core/Neo4jTemplate.java

```diff
@@ -410,9 +412,9 @@ private Object convertIdValues(@Nullable Neo4jPersistentProperty idProperty, Obj
 		}
 
 		class Tuple3<T> {
-			T t1;
-			boolean t2;
-			T t3;
+			final T t1;
+			final boolean t2;
```

Comment:
Because it got touched again, can we please introduce some proper names for the fields in the tuple?